lang/rust/avro/src/schema.rs

```diff
@@ -1282,6 +1280,23 @@ impl Parser {
         Ok(schema)
     }
 
+    fn validate_name(
+        &self,
+        name: &String
+    ) -> bool {
+        let i1 = name.len();
+        if i1 == 0 {
+            return false;
+        }
+        let mut result : bool = name.chars().nth(0).unwrap().is_alphabetic();
+
+        let size = name.chars().count();
+        for i in 1..size {
```

Comment:
no need to iterate if `result` is `false`

Reply:
ok